beginning.md

```diff
@@ -79,18 +83,18 @@ Anything written afer *#* will be ignored.
 
 It is a multi line string
 Have fun, when we print it
-
 >>> 
 ```
-* We can *print* method to display the output. 
+* We can `print` method to display the output.
 
-## Whitespaces and Identation
-* We gives whitespaces between operators to make code more readable.
+## Whitespaces and Indentation
+* We give whitespaces between operators to make the code more readable.
```

Comment:
white spaces needs not only with operators but also while defining list and dict.